src/components/IOUConfirmationList.js

```diff
@@ -131,9 +142,13 @@ class IOUConfirmationList extends Component {
                 indexOffset: 0,
             });
         } else {
-        // $ Should be replaced by currency symbol once available
             const formattedParticipants = getIOUConfirmationOptionsFromParticipants(this.props.participants,
-                `$${this.props.iouAmount}`);
+                this.props.translate('iou.request', {
```

Comment:
Same here, we shouldn't use the `Request [currency]` string here, just `[currency]`

![Screenshot 2021-05-20 at 12 30 06](https://user-images.githubusercontent.com/10736861/118971737-af5fe700-b967-11eb-81c8-dbd9c3fcdfc2.png)